packages/mermaid/src/diagrams/sequence/sequenceDb.js

```diff
@@ -167,6 +215,44 @@ export const parseMessage = function (str) {
   return message;
 };
 
+// We expect the box statement to be color first then description
+// The color can be rgb,rgba,hsl,hsla, or css code names  #hex codes are not supported for now because of the way the char # is handled
+// We extract first segment as color, the rest of the line is considered as text
+export const parseBoxData = function (str) {
+  const match = str.match(/^((?:rgba?|hsla?)\s*\(.*\)|\w*)(.*)$/);
+  let color = match != null && match[1] ? match[1].trim() : 'transparent';
+  let title = match != null && match[2] ? match[2].trim() : undefined;
+
+  // check that the string is a color
+  if (window && window.CSS) {
+    if (!window.CSS.supports('color', color)) {
+      color = 'transparent';
+      title = str.trim();
+    }
+  } else {
+    var s = new Option().style;
+    s.color = color;
+    if (s.color !== color) {
+      color = 'transparent';
+      title = str.trim();
+    }
+  }
+
+  const boxData = {
+    color: color,
+    text: title !== undefined ? title.replace(/^:?(?:no)?wrap:/, '') : undefined,
```

Comment:
I think the input str could use some sanitization using the sanitizeText function.

Reply:
Done in https://github.com/mermaid-js/mermaid/pull/3965/commits/7b5bebd9c4d1cd2144dac1e050ab1aa8c9a61d9d